### DISCUSSION TITLE
[libsrtp]How to enable openssl support in libsrtp?

### DISCUSSION THREAD

**Discussion by pedoc** [microsoft/vcpkg#30380] [Q&A]
Is there a switch to make libsrtp work with openssl? Thanks

**Comment by autoantwort** [comment]
Currently not

**Reply by pedoc** [reply]
![image](https://user-images.githubusercontent.com/6604230/227215287-fcdbd174-0a3e-404d-aa79-26b670875b2e.png)

it's worked
